server/src/main/java/kr/codesquad/library/domain/rental/firstclass/Rentals.java

```diff
@@ -0,0 +1,25 @@
+package kr.codesquad.library.domain.rental.firstclass;
+
+import kr.codesquad.library.domain.book.Book;
+import kr.codesquad.library.domain.rental.Rental;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+public class Rentals {
+
+    private final List<Rental> rentals;
+
+    public Rentals(List<Rental> rentals) {
+        this.rentals = rentals;
+    }
+
+    public Rental find(Book book) {
+        return rentals.isEmpty() ? null : rentals.get(drawByBook(book) - 1);
```

Comment:
`Nit`

저도 3항 연산자의 사용을 선호하지만 가독성을 해치는 경우가 많습니다.

```suggestion
        return rentals.isEmpty() 
        ? null 
        : rentals.get(drawByBook(book) - 1);
```